docs/index.md

```diff
@@ -0,0 +1,39 @@
+(_proxy_py_index)=
+# {{ project }} Documentation
+
+```{spelling}
+acceptor
+acceptors
+Changelog
+decrypt
+decrypted
+discoverable
+DNS
+Facebook
+http
+macOS
+Pre
+Py
+Scalable
+```
+
+```{include} ../README.md
+:end-before: (DO-NOT-REMOVE-docs-badges-START)
+```
+
+```{include} ../README.md
+:start-after: (DO-NOT-REMOVE-docs-badges-END)
+```
+
+```{toctree}
+:hidden:
+
+Glossary <glossary>
```

Comment:
This syntax allows choosing a custom name for the menu item. In most cases, you'd just type in the document name and its title would be used here.